velox/core/PlanNode.cpp

```diff
@@ -899,6 +901,7 @@ folly::dynamic UnnestNode::serialize() const {
   if (withOrdinality_) {
```

Comment:
Please also update void UnnestNode::addDetails(std::stringstream& stream) const {, it will be used in PlanNode->toString

Reply:
done